ScnSoft-Education/LeetCodeTraining.Tests/StringToInteger8.Tests.cs

```diff
@@ -1,5 +1,5 @@
 ﻿using LeetCodeTraining.StringToInteger8;
-namespace LeetCodeTraining.Tests
+namespace StringToIntegerTests
```

Comment:
fix namespace, it should be as "LeetCodeTraining.Tests.[FileName]"

Reply:
fixed errors in namespace